specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-01-01/apimapis.json

```diff
@@ -2618,6 +2618,885 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/issues": {
+      "get": {
+        "tags": [
+          "ApiIssues"
+        ],
+        "operationId": "ApiIssues_ListByService",
+        "description": "Lists all issues assosiated with the specified API.",
+        "x-ms-examples": {
+          "ApiManagementListApiIssues": {
+            "$ref": "./examples/ApiManagementListApiIssues.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "| Field       | Supported operators    | Supported functions               |\n|-------------|------------------------|-----------------------------------|\n| id          | ge, le, eq, ne, gt, lt | substringof, startswith, endswith |\n| state        | eq                     |                                   |\n| userId          | ge, le, eq, ne, gt, lt | substringof, startswith, endswith |"
```

Comment:
Please fix the description to be more like the docs in the code.

Reply:
this will be shown as a table on the HTML doc page. Like here: https://docs.microsoft.com/en-us/rest/api/apimanagement/apidiagnostic/listbyservice#uri-parameters ($filter parameter). Or did I miss the point?

Reply:
I was worried this will look bad in generated SDKs. But I just confirmed with AutoRest owners this will be handled correctly. So I'm signing off.